config/routes.rb

```diff
@@ -6,4 +6,5 @@
   get '/seances', to: 'seances#index'
   get '/tickets', to: 'tickets#index'
   get '/reservations', to: 'reservations#index'
+  get '/create_hall', to: 'halls#new'
```

Comment:
https://github.com/lukasz-ligizynski/Cinema/pull/12#discussion_r804406703

Reply:
Got it :D I will send a later new version of all routes